main.js

```diff
@@ -160,4 +160,155 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+let typesOfTvs = inventory.map((televisions) => {
+  return televisions.type
+})
+
+function soldOutTvs(inventory) {
+  let filterSoldOutTvs = inventory.filter((televisions) => {
+    if (televisions.sold === televisions.originalStock) {
+      return true
+    } else {
+      return false
+    }
+  })
```

Comment:
Omdat `televisions.sold === televisions.originalStock` op zichzelf al een boolean is kun je deze ook direct teruggeven:
```
let soldOutTelevisions = inventory.filter(television => television.sold === television.originalStock);
```